ruleset.xml

```diff
@@ -1,5 +1,137 @@
 <?xml version="1.0"?>
-<ruleset name="cgl-typo3">
+<ruleset name="CGL-TYPO3">
     <!-- Base rules -->
-    <rule ref="cgl-php"/>
+    <rule ref="CGL-PHP"/>
+
+    <!-- Exclude third party code -->
+    <exclude-pattern>*/.git/*</exclude-pattern>
+    <exclude-pattern>*typo3/contrib/*</exclude-pattern>
+    <exclude-pattern>*typo3/sysext/adodb/adodb/*</exclude-pattern>
```

Comment:
Why do we not exclude the whole `typo3` folder but some sub-sub folders and files?

Or even, instead of excluding so mach, just lint the stuff we want, which should basically just be `localPackages` and some configuration files like `AdditionalConfiguration.php`?